### PR TITLE
ERC-5006: Change Status to Review

### PR DESCRIPTION
**Pull request by emojidao**
ERC-5006: Change Status to Review

### CODE REVIEW

**Review by 0xanders — APPROVED** [review]

File: EIPS/eip-5006.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: ERC-1155 Usage Rights Extension
 description: Add a user role with restricted permissions to ERC-1155 tokens.
 author: Lance (@LanceSnow), Anders (@0xanders), Shrug <shrug@emojidao.org>
 discussions-to: https://ethereum-magicians.org/t/eip5006-erc-1155-usage-rights-extension/8941
-status: Draft
+status: Review
```
Comment (Pandapip1) [inline EIPS/eip-5006.md:7]: I can't comment on unchanged portions. Here are some things that need changing:

- L3: Do not include the EIP number in the title
- L4: Change ERC-1155 to EIP-1155.
- L11: 721 needs to be added (see L20)
- L16: A link to EIP-1155 needs to be provided (and needs to be referenced as an EIP):
  
  `ERC-1155 -> [EIP-1155](./eip-1155.md)`
- L20: Add a link to EIP-721. A link to EIP-1155 isn't necessary (but feel free to include one)
  
  `ERC-721 -> [EIP-721](./eip-721.md)`
- L32: The `UpdateUser` event should have some NatSpec.
- L78: I would recommend using multi-line comments for consistency.
- L97: I would rephrase the last sentence as:
  
  `The following are some problems that are solved by this standard:`
- L101: There is a double space and there are two contractions (change them to "it is")
- L109: Compatible with what?
- L111: Mention that there are no conflicts between EIP-5006 and EIP-1155
- L339: Please elaborate. It is currently unclear what this means.
- L148-L335: I would recommend moving the reference implementation and test cases out of inline and linking to their copies in the assets folder.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Here is my first pass.

File: EIPS/eip-5006.md (hunk 1 above)
Comment (SamWilsn) [reply EIPS/eip-5006.md:7]: I think `EIP-1155` is fine in the title, since this is an extension.

Comment (Pandapip1) [reply EIPS/eip-5006.md:7]: > I think EIP-1155 is fine in the title, since this is an extension.

For some reason, I thought that the title originally contained the actual EIP number (5006). Must have misread it!
